demos/embeddings/barbell-graphwave.ipynb

```diff
@@ -0,0 +1,162 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "classes = [0,] * len(gnx.nodes)\n",
+    "classes[m1 - 1] = 1\n",
+    "classes[+m2] = 1\n",
+    "classes[m1 : (m1 + m2)] = list(abs(x) + 2 for x in range(m2 // 2, -m2 // 2, -1))\n",
```

Comment:
Per the printing above, this results in a "funny" ordering, that doesn't match the paper. I think if we try to flip it to be `[0, 0, ..., 0, 1, 2, 3, 4, 5, 6, 7, 6, ..., 2, 1, 0, 0, ... 0]`, it'll look closer to the paper.

One possibility for doing this is to use `range`:

```python
classes = [0,] * len(nodes)
# number of nodes with a non-zero class (the path, plus the nodes it connects to on either end)
nonzero = m2 + 2
# count up to the halfway point (rounded up)
first = range(1, (nonzero + 1) // 2 + 1)
# and down for the rest
second = reversed(range(1, nonzero - len(first) + 1))
classes[m1 - 1:(m1 + m2) + 1] = list(first) + list(second)
assert len(classes) == len(nodes)
```

The graph and embedding plots then look like:

![image](https://user-images.githubusercontent.com/1203825/74703301-dc9f8000-5260-11ea-939f-93d47c60bbc5.png)
![image](https://user-images.githubusercontent.com/1203825/74703315-e1fcca80-5260-11ea-981d-d85c427a626b.png)

which are qualitatively more similar to the paper.